hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCommitLocks.java

```diff
@@ -184,7 +184,7 @@ public void testLockAcquisitionAfterRetries() throws TException, InterruptedExce
 
     spyOps.doCommit(metadataV2, metadataV1);
 
-    Assert.assertEquals(1, spyOps.current().schema().columns().size()); // should be 1 again
+    Assertions.assertThat(spyOps.current().schema().columns()).hasSize(1); // should be 1 again
```

Comment:
those can probably be statically imported here

Reply:
Added